api/src/main/java/org/openmrs/OrderType.java

```diff
@@ -26,10 +26,6 @@
 	
 	public static final long serialVersionUID = 23232L;
 	
-	public static final String DRUG_ORDER_TYPE_UUID = "131168f4-15f5-102d-96e4-000c29c2a5d7";
-	
-	public static final String TEST_ORDER_TYPE_UUID = "52a447d3-a64a-11e3-9aeb-50e549534c5e";
-	
```

Comment:
I don't think we actually want to *remove* these constants. Since they are *public*, this would be a potentially breaking change.

Reply:
Thanks @ibacher  for the oversight into this .Was actually wondering what would happen if they are removed .but since you highlighted they're public  .It's wise that way ..Let me  look into that

Reply:
> I don't think we actually want to _remove_ these constants. Since they are _public_, this would be a potentially breaking change.

I had dealt with the dependents given their static nature , but let me have them restored.